rllib_contrib/apex_dqn/examples/apex_dqn_cartpole_v1.py

```diff
@@ -0,0 +1,58 @@
+import argparse
```

Comment:
why "v1"?

Reply:
we are using the cartpole v1 environment